<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteProfilesPage</class>
 <widget class="QDialog" name="DeleteProfilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="DeleteProfile_1">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>60</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteProfile_2">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>100</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteProfile_3">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>140</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteProfile_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteProfile_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_1">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>64</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_2">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>104</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_3">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>145</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_4">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>185</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_5">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>224</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
